pytorch_lightning/utilities/argparse_utils.py

```diff
@@ -2,8 +2,6 @@
 
 rank_zero_deprecation("`argparse_utils` package has been renamed to `argparse` since v1.2 and will be removed in v1.4")
 
```

Comment:
Why did you leave this?

Reply:
because the module itself is deprecated and needs to stag because of checkpoint compatibility

Reply:
We need the file but not the deprecation message

If you want to keep the deprecation message until the file can be completely removed, we should also keep the deprecation message test

Reply:
added back b36c5c849282aca46bf26ad94525de3ad271cb9a